e2e/testsuite/testsuite.go

```diff
@@ -159,6 +161,39 @@ func (s *E2ETestSuite) SetupChainsRelayerAndChannel(ctx context.Context, channel
 	return r, chainAChannels[len(chainAChannels)-1]
 }
 
+// generatePathName generates the path name using the test suites name
+func (s *E2ETestSuite) generatePathName() string {
+	pathName := fmt.Sprintf("%s-path-%d", s.T().Name(), s.pathNameIndex)
+	s.pathNameIndex++
+	return strings.ReplaceAll(pathName, "/", "-")
```

Comment:
for every endpoint (client/connection/channel) stack we need a new path name, so I added an index kept by the `E2ETestSuite`